contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,202 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+Author(s): [Ronak Lakhotia](https://github.com/RonakLakhotia)
+
+## What Is Angular?
+
+Angular is an open-source frontend Javascript framework that helps you build <tooltip content="Web apps that are more responsive to user interactions ">Single Page Web Applications</tooltip>.
```

Comment:
This tooltip does not define the term correctly.

Reply:
Do you need to mention SPA in this sentence at all? Angular is only for SPA-type Web applications?

Reply:
Not only SPAs, using Angular for SPA is a use case. Can it be rephrased to `interactive we applications`?

Reply:
As opposed to _non-interactive web applications_? :-p

Reply:
`Angular is an open-source frontend Javascript framework that helps you build web applications` seems most appropriate but it seems too general for a starting introduction maybe? :P

Reply:
It is what it is :-)
In fact, `Angular is one of the popular Javascript frameworks used for building web applications.`? The word `popular` justifies why this chapter exists. The `open-source` is not a key feature important enough to be mentioned in the opening sentence.

Reply:
Yeah this makes more sense. 👍